src/content/cloud/okteto-cli.mdx

```diff
@@ -102,6 +121,6 @@ The installation job will run in a Debian Linux container with the following too
 - `openssh`
 - `wait-for-it`
 
-The installation job is automatically configured with your Okteto and Kubernetes credentials.
+You can also configure your own [container image](reference/manifest.mdx/#deploy-remotely) to excute your `deploy` commands.
```

Comment:
It's not clear why this is useful. Maybe add that this is the way to include other tools?

Reply:
Right, I will explain the use case